tst/SdpApiTest.cpp

```diff
@@ -228,6 +228,168 @@ TEST_F(SdpApiTest, setTransceiverPayloadTypes_HasRtxType) {
     doubleListFree(pTransceivers);
 }
 
+TEST_F(SdpApiTest, populateSingleMediaSection_SendRecvDirection) {
+
+    auto sessionDescriptionNoMedia = R"(v=2
+o=- 1904080082932320671 2 IN IP4 127.0.0.1
+s=-
+t=0 0
+a=group:BUNDLE 0 1
+a=msid-semantic: WMS f327e13b-3518-47fc-8b53-9cf74d22d03e
+)";
+    SessionDescription remoteSessionDescription, localSessionDescription;
+    MEMSET(&localSessionDescription, 0x00, SIZEOF(SessionDescription));
+    MEMSET(&remoteSessionDescription, 0x00, SIZEOF(SessionDescription));
+    EXPECT_EQ(serializeSessionDescription(&remoteSessionDescription, (PCHAR) sessionDescriptionNoMedia), STATUS_SUCCESS);
+
+    PHashTable pCodecTable;
+    PHashTable pRtxTable;
+    KvsRtpTransceiver transceiver;
+
+    RtcConfiguration configuration;
+    PRtcPeerConnection offerPc = NULL;
+
+//    // Create peer connection
+    EXPECT_EQ(createPeerConnection(&configuration, &offerPc), STATUS_SUCCESS);
+    PKvsPeerConnection KvsPeerConnection = (PKvsPeerConnection) (offerPc);
+    KvsPeerConnection->isOffer = TRUE;
+
+    transceiver.sender.track.codec = RTC_CODEC_H264_PROFILE_42E01F_LEVEL_ASYMMETRY_ALLOWED_PACKETIZATION_MODE;
+    transceiver.transceiver.direction = RTC_RTP_TRANSCEIVER_DIRECTION_SENDRECV;
+    transceiver.sender.packetBuffer = NULL;
+    transceiver.sender.retransmitter = NULL;
+
+    EXPECT_EQ(STATUS_SUCCESS, hashTableCreate(&(KvsPeerConnection->pCodecTable)));
+    EXPECT_EQ(STATUS_SUCCESS, hashTablePut(KvsPeerConnection->pCodecTable, RTC_CODEC_H264_PROFILE_42E01F_LEVEL_ASYMMETRY_ALLOWED_PACKETIZATION_MODE, 1));
+    EXPECT_EQ(STATUS_SUCCESS, hashTableCreate(&(KvsPeerConnection->pRtxTable)));
+    EXPECT_EQ(STATUS_SUCCESS, hashTablePut(KvsPeerConnection->pRtxTable, RTC_RTX_CODEC_H264_PROFILE_42E01F_LEVEL_ASYMMETRY_ALLOWED_PACKETIZATION_MODE, 2));
+
+    EXPECT_EQ(STATUS_SUCCESS, doubleListCreate(&(KvsPeerConnection->pTransceievers)));
+    EXPECT_EQ(STATUS_SUCCESS, doubleListInsertItemHead(KvsPeerConnection->pTransceievers, (UINT64) (&transceiver)));
+    EXPECT_EQ(STATUS_SUCCESS, setTransceiverPayloadTypes(KvsPeerConnection->pCodecTable, KvsPeerConnection->pRtxTable, KvsPeerConnection->pTransceievers));
+
+
+    STATUS retStatus = populateSessionDescription(KvsPeerConnection, &remoteSessionDescription, &localSessionDescription);
```

Comment:
To populate use `createOffer` instead, then `deserialize`. 

It will just make the tests easier to maintain long term if using the public APIs.

Reply:
Done